src/App/Tests/Controller/DefaultControllerTest.php

```diff
@@ -36,8 +36,7 @@ public function createApplication()
     public function testIndex()
     {
         $client = $this->createClient();
-        $client->followRedirects();
-        $crawler = $client->request('GET', '/');
+        $crawler = $client->request('GET', '/', array(), array(), array('HTTPS' => true));
 
```

Comment:
I had redirects to https on each call. So I switched for the tests to https.
I can remove it or change the travis-config to 

`$config['requireHttps'] = true;`